test/ios/App-Info.plist

```diff
@@ -34,6 +36,8 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>MGLMapboxMetricsEnabledSettingShownInApp</key>
+	<true/>
```

Comment:
Do you mean adding an actual button and KIF-pushing it? It seems like [`[MetricsTests testOptOutAtRuntimeDisablesMetrics]`](https://github.com/mapbox/mapbox-gl-native/blob/dfd6ff00a239ef7c4762346edf798aa5dfb7bf3f/test/ios/MetricsTests.m#L262-L279) covers that case, sans UI.